chore(quo): add profile/extended collectible component

J-Son89 opened the pull request:
fixes: https://github.com/status-im/status-mobile/issues/19286

Designs: https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=34752-33684&mode=design&t=PPSuhaLdq7QTrIjW-4

This pr adds the extended collectible component needed for Multiple NFT's feature in the wallet.

<img width="300px" src="https://github.com/status-im/status-mobile/assets/22799766/f9ed197e-e5e9-47f4-81a8-c2ddff47976b" />

<img width="300px" src="https://github.com/status-im/status-mobile/assets/22799766/61c2318e-8104-436a-b495-21ef0df784dd" />

<img width="300px" src="https://github.com/status-im/status-mobile/assets/22799766/19afa342-de11-49f7-9d01-3627a94efd7b" />

<img width="300px" src="https://github.com/status-im/status-mobile/assets/22799766/c1c93208-2d8e-4712-9bb5-9e742bacbe41" />

Testing Notes: 
This only adds a design system component, no other functionality is added here.

J-Son89 commented on src/quo/components/profile/expanded_collectible/view.cljs:
```diff
@@ -0,0 +1,54 @@
+  (let [[image-size set-image-size] (rn/use-state {})]
+    (rn/use-effect
+     (fn []
+       (p/let [[image-width image-height] (rn/image-get-size image-src)]
```
getting the image to match the designs is tricky because we need to have the correct dimension of the image. 
By getting the image dimensions we can set the appropriate aspect ratio and from there create a container that perfectly wraps the image, and with that place the counter in a consistent position relative to the corner of the image regardless of it's size.

OmarBasem replied:
Shouldn't we not be using `use-effect` anymore? This one seems like a valid use case, but maybe also check with @flexsurfer

flexsurfer replied:
yeah seems like image-size state and effect are redundant, it could be just calculated directly in let

J-Son89 replied:
Was getting some weird errors trying that because that method is async.
@clauxx any ideas about this?

flexsurfer replied:
oh it's `p/let` didn't notice that, can we do it more visible?

clauxx replied:
@flexsurfer if we put it inside let, it will be called on every re-render, no?

clauxx replied:
we could also put it inside `use-mount` if that helps, unless the props image can change

flexsurfer replied:
yes i didn't notice the promise, seems like use-effect is legit here

J-Son89 replied:
Yeah the image source can change in theory.

clauxx replied:
> can we do it more visible?

I guess replacing `p/let` with `p/then`/`.then` to make it clearer it's a promise could help

J-Son89 replied:
imo, just changing the alias to promesa.

e.g `promesa/let`

clauxx replied:
no strong preference from my side

J-Son89 replied:
ok for the time being I would prefer that we alias as `promesa`. Hopefully it will be clearer then for other developers who are unfamiliar with the library/ to look out for this.

I think using `/let` is more succinct and it would be good to have some different examples of using promesa in the codebase for other developers to work from.

J-Son89 replied:
going to do this in a separate pr to avoid the noise, has to be made across all the repo 👍

J-Son89 replied:
going to do this in a separate pr as I have to update the consistent alias config too

J-Son89 commented on src/react_native/core.cljs:
```diff
@@ -40,7 +40,8 @@
                         uri
-                        (fn [width height] (res [width height]))
+                        (fn [width height]
+                          (res [width height]))
```
will revert

J-Son89 commented on src/status_im/common/resources.cljs:
```diff
@@ -110,6 +110,7 @@
    :decentraland          (js/require "../resources/images/mock2/decentraland.png")
    :diamond               (js/require "../resources/images/mock2/diamond.png")
    :gif                   (js/require "../resources/images/mock2/gif.png")
+   :horizontal            (js/require "../resources/images/mock2/horizontal.png")
```
revert

J-Son89 commented on src/status_im/common/resources.cljs:
```diff
@@ -126,7 +127,8 @@
    :user-picture-male4    (js/require "../resources/images/mock2/user_picture_male4.png")
-   :user-picture-male5    (js/require "../resources/images/mock2/user_picture_male5.png")})
+   :user-picture-male5    (js/require "../resources/images/mock2/user_picture_male5.png")
+   :vertical              (js/require "../resources/images/mock2/vertical.png")})
```
revert

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | 139cb7c4 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-19297/1/) | 2024-03-18 01:18:19 | ~4 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-19297/1/consoleText) |

clauxx commented on src/quo/components/profile/expanded_collectible/view.cljs:
```diff
@@ -0,0 +1,54 @@
+     [rn/view {:style {:flex 1}}
+      [rn/image
+       {:style  (merge (shadows/get 2)
+                       {:width         "100%"
```
we shouldn't use % dimensions

J-Son89 replied:
I'm quite confident that to make the image stretch accordingly you need to set width 100% and an aspect ratio so that the height stretches out respective to that.

vkjr replied:
Should we move this to style and maybe add the reasoning for `width "100%"` as a comment there?

J-Son89 replied:
sorry yeah moved to style folder! 🙏 
I'll add a comment too

smohamedjavid commented on src/quo/components/profile/expanded_collectible/view.cljs:
```diff
@@ -0,0 +1,54 @@
+                          :height       image-height
+                          :aspect-ratio (/ image-width image-height)})))
+     [image-src])
+    ^{:key (str "expanded-collectible-" image-src)}
```
Quick question (maybe a dumb one): Why do we need this key? There is no loop, and React automatically takes care of it. If we need this key, can we place this key along with other props of `pressable`?

J-Son89 replied:
yeah I can see about removing this now. It was previously needed so that the dimensions of the image would update if the image-src changes. However I think my current approach does this anyway. 
Thanks for the reminder 👍

J-Son89 replied:
removed, thanks!

J-Son89 committed:
chore(quo): add profile/extended collectible component

J-Son89 commented:
I will skip QA here as it's just a design system component being added here.
@Francesca-G could I get a design review please 🙏 
cc @pavloburykh

status-im-auto commented:
## 92% of end-end tests have passed
```
Total executed tests: 48
Failed tests: 3
Expected to fail tests: 1
Passed tests: 44
```
```
IDs of failed tests: 702777,702855,702731 
```
```
IDs of expected to fail tests: 703503 
```
<h3>Failed tests (3)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&PR_ID=19297&TR_CASE_IDS=702777,702855,702731">Rerun failed tests</a></li><br/><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504552">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `xpath`: `//*[@text="Paste"]` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
activity_center/test_activity_center.py:226: in test_add_contact_field_validation
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Error message is not shown for value "cs3:X1cQV8s:DfL:t9yuHZHxYpSu21cHHJzhUsxHnUR3LhE2pTLM4DttVgDb:pUPdvKkMEt93bpRJw6p18fdofY9LQDkFUrDYMJtBxpP"
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/657a931c9f174e249223153b0d8f7902?auth=87c752b637944a7f9e4a0da00ee9a776#141">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab59c70416c4417eb9c9f14f1d6a601b?auth=74f58a6f2a8f64ed7bcbf48316ad3e5a#180">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504571">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//*[starts-with(@text,'Message before edit 1-1')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 2: `Text` is `Sent` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:379: in test_1_1_chat_edit_message
    self.chat_2.chat_element_by_text(message_before_edit_1_1).wait_for_status_to_be("Delivered")
../views/chat_view.py:243: in wait_for_status_to_be
    raise TimeoutException("Message status was not changed to %s, it's %s" % (expected_status, current_status))
 Message status was not changed to Delivered, it's Sent
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4cdb367fce614df885eadd0026228116?auth=ec1332d2aa2a0d4688b28ae3e8b5b376#406">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/99d7bf07e17747eeacefb94adb592989?auth=4560e6d24e54f1f3fed61c2331bb0ad6#289">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504546">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `xpath`: `//*[starts-with(@text,'Message 3')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 1: `Text` is `Sent` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:222: in test_1_1_chat_pin_messages
    self.chat_1.chat_element_by_text(message).wait_for_status_to_be("Delivered")
../views/chat_view.py:243: in wait_for_status_to_be
    raise TimeoutException("Message status was not changed to %s, it's %s" % (expected_status, current_status))
 Message status was not changed to Delivered, it's Sent
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4cdb367fce614df885eadd0026228116?auth=ec1332d2aa2a0d4688b28ae3e8b5b376#202">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/99d7bf07e17747eeacefb94adb592989?auth=4560e6d24e54f1f3fed61c2331bb0ad6#167">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504589">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Curated communities not loading, https://github.com/status-im/status-mobile/issues/17852]]</b><br/><br/></td></tr></tbody></table></details><h3>Passed tests (44)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504553">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4cdb367fce614df885eadd0026228116?auth=ec1332d2aa2a0d4688b28ae3e8b5b376#140">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/99d7bf07e17747eeacefb94adb592989?auth=4560e6d24e54f1f3fed61c2331bb0ad6#122">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504548">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4cdb367fce614df885eadd0026228116?auth=ec1332d2aa2a0d4688b28ae3e8b5b376#485">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/99d7bf07e17747eeacefb94adb592989?auth=4560e6d24e54f1f3fed61c2331bb0ad6#407">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504560">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4cdb367fce614df885eadd0026228116?auth=ec1332d2aa2a0d4688b28ae3e8b5b376#351">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/99d7bf07e17747eeacefb94adb592989?auth=4560e6d24e54f1f3fed61c2331bb0ad6#254">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504550">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4cdb367fce614df885eadd0026228116?auth=ec1332d2aa2a0d4688b28ae3e8b5b376#293">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/99d7bf07e17747eeacefb94adb592989?auth=4560e6d24e54f1f3fed61c2331bb0ad6#200">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504586">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4cdb367fce614df885eadd0026228116?auth=ec1332d2aa2a0d4688b28ae3e8b5b376#422">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/99d7bf07e17747eeacefb94adb592989?auth=4560e6d24e54f1f3fed61c2331bb0ad6#347">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504545">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4cdb367fce614df885eadd0026228116?auth=ec1332d2aa2a0d4688b28ae3e8b5b376#86">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/99d7bf07e17747eeacefb94adb592989?auth=4560e6d24e54f1f3fed61c2331bb0ad6#87">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504570">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/657a931c9f174e249223153b0d8f7902?auth=87c752b637944a7f9e4a0da00ee9a776#101">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab59c70416c4417eb9c9f14f1d6a601b?auth=74f58a6f2a8f64ed7bcbf48316ad3e5a#100">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504569">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/657a931c9f174e249223153b0d8f7902?auth=87c752b637944a7f9e4a0da00ee9a776#59">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab59c70416c4417eb9c9f14f1d6a601b?auth=74f58a6f2a8f64ed7bcbf48316ad3e5a#58">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504575">test_navigation_jump_to</a>, id: 702936 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eaf50b706ce64d9084ff8111393f238c?auth=3bdf1df4881ecaf81093a050649a50f1#139">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94d94dad9b6040afa7d44e234101bd80?auth=2abc29ee3bc6596c98f471498cd8f385#122">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504576">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eaf50b706ce64d9084ff8111393f238c?auth=3bdf1df4881ecaf81093a050649a50f1#167">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94d94dad9b6040afa7d44e234101bd80?auth=2abc29ee3bc6596c98f471498cd8f385#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504547">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ef1eab57650b41fc82633f5d5c01b16a?auth=39e0dcae2ad91157c2a5a33202a0c300#343">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ff94cd1dfdf240dc9610d3a44ab14720?auth=2d59d6202180f085087d19d45edd3c3b#246">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3094fbe1359e440d8bb98bb62132e0e1?auth=6fc0d99293a3ffbefd14e49a6b14eb55#241">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504587">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ef1eab57650b41fc82633f5d5c01b16a?auth=39e0dcae2ad91157c2a5a33202a0c300#471">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ff94cd1dfdf240dc9610d3a44ab14720?auth=2d59d6202180f085087d19d45edd3c3b#292">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3094fbe1359e440d8bb98bb62132e0e1?auth=6fc0d99293a3ffbefd14e49a6b14eb55#259">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504584">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ef1eab57650b41fc82633f5d5c01b16a?auth=39e0dcae2ad91157c2a5a33202a0c300#262">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ff94cd1dfdf240dc9610d3a44ab14720?auth=2d59d6202180f085087d19d45edd3c3b#179">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3094fbe1359e440d8bb98bb62132e0e1?auth=6fc0d99293a3ffbefd14e49a6b14eb55#148">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504583">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ef1eab57650b41fc82633f5d5c01b16a?auth=39e0dcae2ad91157c2a5a33202a0c300#156">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ff94cd1dfdf240dc9610d3a44ab14720?auth=2d59d6202180f085087d19d45edd3c3b#109">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3094fbe1359e440d8bb98bb62132e0e1?auth=6fc0d99293a3ffbefd14e49a6b14eb55#89">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504557">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ef1eab57650b41fc82633f5d5c01b16a?auth=39e0dcae2ad91157c2a5a33202a0c300#137">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ff94cd1dfdf240dc9610d3a44ab14720?auth=2d59d6202180f085087d19d45edd3c3b#81">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3094fbe1359e440d8bb98bb62132e0e1?auth=6fc0d99293a3ffbefd14e49a6b14eb55#81">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504558">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ef1eab57650b41fc82633f5d5c01b16a?auth=39e0dcae2ad91157c2a5a33202a0c300#306">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ff94cd1dfdf240dc9610d3a44ab14720?auth=2d59d6202180f085087d19d45edd3c3b#220">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3094fbe1359e440d8bb98bb62132e0e1?auth=6fc0d99293a3ffbefd14e49a6b14eb55#207">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504591">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/81cbd3ca270c4dbd935c4bfb73a6b283?auth=f5eaf5275baf4b900f5d8b0f401ee98a#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504551">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/81cbd3ca270c4dbd935c4bfb73a6b283?auth=f5eaf5275baf4b900f5d8b0f401ee98a#199">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504582">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#212">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#167">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504572">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#241">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#233">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504563">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#291">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#289">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504580">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#556">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#434">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504574">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#397">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#355">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504592">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#595">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#462">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504562">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#195">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#138">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504561">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#153">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504566">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#315">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#304">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504565">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#171">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#126">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504564">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0d2e16e880649fdba51a6bd23583dac?auth=aae61673d988b3940f6656300d679f41#360">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88ec9da7654d4c439e88a8b40f3d4027?auth=718766e7c910f17f4271ab306af0cacc#345">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504581">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/11f3721e251d4bbf9c92017bb0b12bc5?auth=0b6b42a018a48e4ae21e63389e1f3528#252">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504549">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/11f3721e251d4bbf9c92017bb0b12bc5?auth=0b6b42a018a48e4ae21e63389e1f3528#79">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504573">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/11f3721e251d4bbf9c92017bb0b12bc5?auth=0b6b42a018a48e4ae21e63389e1f3528#108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504568">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/11f3721e251d4bbf9c92017bb0b12bc5?auth=0b6b42a018a48e4ae21e63389e1f3528#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504585">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/11f3721e251d4bbf9c92017bb0b12bc5?auth=0b6b42a018a48e4ae21e63389e1f3528#127">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504578">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e38cff2969f144999c198f2e3112b59e?auth=3173339152cf474093ca9725aeec6807#119">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c5c6951df15a4e2ebbe37732832266ab?auth=ac04e76a18acc00477992573e73eaa91#103">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504579">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e38cff2969f144999c198f2e3112b59e?auth=3173339152cf474093ca9725aeec6807#167">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c5c6951df15a4e2ebbe37732832266ab?auth=ac04e76a18acc00477992573e73eaa91#119">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504559">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/35b709c8588e449c84b2ee3441f7fc44?auth=cc72ced30efe5efa768c0558d47ba6a8#190">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/617974eb020e45109160f8c69fdc7f02?auth=8bd3792f6f108cb1d5d3c021e915fb8f#177">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504577">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/35b709c8588e449c84b2ee3441f7fc44?auth=cc72ced30efe5efa768c0558d47ba6a8#331">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/617974eb020e45109160f8c69fdc7f02?auth=8bd3792f6f108cb1d5d3c021e915fb8f#337">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504556">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/35b709c8588e449c84b2ee3441f7fc44?auth=cc72ced30efe5efa768c0558d47ba6a8#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/617974eb020e45109160f8c69fdc7f02?auth=8bd3792f6f108cb1d5d3c021e915fb8f#112">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504567">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/35b709c8588e449c84b2ee3441f7fc44?auth=cc72ced30efe5efa768c0558d47ba6a8#328">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/617974eb020e45109160f8c69fdc7f02?auth=8bd3792f6f108cb1d5d3c021e915fb8f#306">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504590">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/35b709c8588e449c84b2ee3441f7fc44?auth=cc72ced30efe5efa768c0558d47ba6a8#542">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/617974eb020e45109160f8c69fdc7f02?auth=8bd3792f6f108cb1d5d3c021e915fb8f#524">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504555">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b6fa0f5cd1e547c7b7025c7b588cea05?auth=6e4683452f774d5fef1e1bdea87544b9#175">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c728810d7abf4245976b2943af76a507?auth=8364dbc0412a86e422b8904fa3b233fa#128">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504554">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b6fa0f5cd1e547c7b7025c7b588cea05?auth=6e4683452f774d5fef1e1bdea87544b9#86">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c728810d7abf4245976b2943af76a507?auth=8364dbc0412a86e422b8904fa3b233fa#87">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3504588">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b6fa0f5cd1e547c7b7025c7b588cea05?auth=6e4683452f774d5fef1e1bdea87544b9#101">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c728810d7abf4245976b2943af76a507?auth=8364dbc0412a86e422b8904fa3b233fa#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

pavloburykh commented:
@J-Son89 thanx for the PR. E2E failures are not PR related. Ready for merge.

J-Son89 committed:
mali

J-Son89 committed:
test correction

Francesca-G reviewed (commented):
here's the [review](https://www.figma.com/file/e9woQowfaIOCBrXS1jgkhZ/Review-Components-%2F-Mobile?type=design&node-id=31%3A151&mode=design&t=SrPatJFQg5nZ0ei7-1) :)

J-Son89 commented:
thanks @Francesca-G - I addressed the comments.

Removed that black border that was there ✅ 
Added the shadow properly ✅ 
The counter border radius looks fine to me, albeit it is a separate component. ❓ 
The height of the vertical image is fine, it's just based on the available space so we can control that from the view that uses this component. ✅ 


If there is an issue there we can do as a follow up anyway as it is older code. Let me know what you think 🙏

Francesca-G commented:
> thanks @Francesca-G - I addressed the comments.
> 
> Removed that black border that was there ✅ Added the shadow properly ✅ The counter border radius looks fine to me, albeit it is a separate component. ❓ The height of the vertical image is fine, it's just based on the available space so we can control that from the view that uses this component. ✅
> 
> If there is an issue there we can do as a follow up anyway as it is older code. Let me know what you think 🙏

Thank you @J-Son89, thanks for checking the border radius and addressing all the other issues.
It looks good to me 👍

J-Son89 committed:
Merge branch 'develop' into jc/expanded-c

J-Son89 committed:
Merge branch 'develop' into jc/expanded-c
